src/services/user.ts

```diff
@@ -0,0 +1,6 @@
+import usersFromServer from '../api/users';
+import { User } from '../types/User';
+
+export function getUserById(userId: number): User | null {
+  return usersFromServer.find(user => user.id === userId) || null;
```

Comment:
Ensure that `usersFromServer` is correctly structured and contains valid user objects with an `id` property to allow the `find` method to work as expected.